templates/repo/diff/box.tmpl

```diff
@@ -230,17 +230,22 @@
 			document.addEventListener('DOMContentLoaded', function() {
 				$('tr.add-code').each(function() {
 					var prev = $(this).prev();
-					if(prev.is('.del-code') && prev.children().eq(3).text().trim() === '') {
-						while(prev.prev().is('.del-code') && prev.prev().children().eq(3).text().trim() === '') {
+					if(prev.is('.del-code') && prev.children().eq(5).text().trim() === '') {
+						while(prev.prev().is('.del-code') && prev.prev().children().eq(4).text().trim() === '') {
```

Comment:
This should probably be 5 not 4 to match the line above.

```suggestion
						while(prev.prev().is('.del-code') && prev.prev().children().eq(5).text().trim() === '') {
```

Reply:
I'm not even sure what this horrible code is trying to achieve, but I'll commit this and test the results.